oci/main.py

```diff
@@ -43,15 +43,66 @@ def run():
              'under "ENGINE_VERSION"',
         default='master')
 
+    stage_parser = subparsers.add_parser(
+        'run',
+        help='run stage for a change')
+    stage_parser.set_defaults(command=run_stage)
+
+    stage_parser.add_argument(
+        '--stage',
+        '-s',
+        help='stage name')
```

Comment:
This is optional argument without a default. If we run this:

    ./ovirt-ci run 12345

We will fail trying to build a stage None.

I think this should be a positional argument like this:

    ./ovirt-ci run check-network 12345

Reply:
I hope i understood correctly.